apps/autonolas-registry/common-util/Contracts/addresses.tsx

```diff
@@ -59,71 +83,60 @@ const COMMON_TEST_ADDRESSES: Addresses = {
 // TODO: add testcases for all networks
 // get addresses from scripts/deployment folder in autonolas-registries repo
 export const ADDRESSES = {
-  1: MAINNET_ADDRESSES,
```

Comment:
just resorted (placed all test nets at the end) and replaced the numbers with [chain.id] for better readability